src/pages/instances/forms/UploadInstanceBackupFileForm.tsx

```diff
@@ -60,19 +62,18 @@ const UploadInstanceBackupFileForm: FC<Props> = ({
   const { hasInstanceImportConversion } = useSupportedFeatures();
 
   const handleSuccess = (instanceName: string) => {
+    const url = `/ui/project/${project?.name}/instance/${instanceName}`;
     const message = (
       <>
-        Created instance <strong>{instanceName}</strong>.
+        Created instance{" "}
+        <ResourceLink type="virtual-machine" value={instanceName} to={url} />.
```

Comment:
This can be a container as well

Reply:
Meaning the type assumption virtual-machine is not correct, this needs to be dynamic I think.

Reply:
Potentially, we can also chain an API call to a successful event and fetch the instance based on the name returned by the operation.

Reply:
Yeah, I think a generic instance type would be fine here and in the other places.